website/docs/components/stepper/list/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,56 @@
+## Usage
+
+### When to use
+
+- In onboarding, account creation, or initial setups that are broken up into multiple steps to more easily segment content.
+- To communicate what has and has not been completed in a multi-step flow.
+- For progress that does not require a user to remain on the current page to complete.
+
+### When not to use
+
+- When the user needs to remain within the step to complete it, use the [Stepper Nav](/components/stepper/nav).
+- For simple forms or when progress doesn't need to be tracked, use structured page content.
+
+## Content
+
+The Stepper List has a title, an optional description, and a generic slot for content.
+
+### Title
+
+Titles provide an overview to help users understand what tasks need to be completed and their purpose within the flow. Titles are required and should be brief.
+
+### Description
+
+Descriptions are optional. They should provide additional context but be brief. We recommend that all steps either have a description or do not.
+
+!!! Do
+
+Ensure description visibility is consistent.
+
+![The stepper with descriptions enabled in every step.](/assets/components/stepper/list/stepper-list-description-do.png)
+
+!!!
+
+!!! Dont
+
+Don't mix and match description visibility.
```

Comment:
```suggestion
Don't mix and match inclusion of descriptions.
```

Reply:
Same as above. There may be other ways to say this but I think using the word "visibility" has more potential for misunderstanding.